### ISSUE TITLE
Read NIDM packs

### ISSUE THREAD

**Issue by cmaumet** [pwilliams1/nidmresults-fslhtml#6]
Could we get the viewer to read nidm packs (i.e. `.nidm.zip`) directly rather than turtle files.

**Comment by pwilliams1** [comment]
How do we want this to work exactly? The viewer accepts the zip file as a command line argument, extracts the turtle file, and then runs as before? Is there anything else in the zip file we need to look at?

**Comment by cmaumet** [comment]
You are right! The only additional thing is that if the NIDM pack was created locally (and not uploaded to NeuroVault) then the location of the files (e.g. the design matrix image) will not be full paths but relative paths (similar to https://github.com/incf-nidash/nidm/blob/master/nidm/nidm-results/fsl/example001/fsl_nidm.ttl#L439), this means you will have to append the full path of the pack to this relative path available in the turtle to retrieve the image.

**Comment by pwilliams1** [comment]
Okay thanks, I'll start having a look at it.

**Comment by pwilliams1** [comment]
Quick question: Do we still want to include the option to specify the turtle file directly, or only handle the zip file case?

**Comment by cmaumet** [comment]
I think it's best if we can keep both. If the file is local then a NIDM pack is more likely to be read if the file is online (e.g. on NeuroVault) then it's more likely to be expanded.
